rssfeed/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.1 on 2017-01-25 13:04
+# Generated by Django 1.9.1 on 2017-01-26 14:47
```

Comment:
This migration is already on develop, can it safely be altered? IE has there been a deploy of this code anywhere yet?